meta-sokol-flex-distro/conf/distro/include/gdbserver-gplv3.inc

```diff
@@ -1,17 +0,0 @@
-# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
I think we need ore information in commit summary for these changes

Reply:
This is already covered in the commit `local.conf.sample: remove binutils and gplv3 hacks`. This is just removing the bits the local.conf was referencing.

Reply:
I also added a mention of this in the commit.